ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml

```diff
@@ -442,6 +444,8 @@ objects:
           ${SSH_PUB_KEY}
         EOF
 
+        cp /tmp/artifacts/installer/install-config.yaml /tmp/artifacts/installer/install-config-bk.yaml
```

Comment:
can you leave a comment why is this needed?

Reply:
I think it was only for testing, we shouldn't save it

Reply:
correct, it was for testing only, removed